docs/docsite/rst/userguide/job_branching.rst

```diff
@@ -3,10 +3,12 @@
 Projects specify the branch, tag, or reference to use from source control in the ``scm_branch`` field. These are represented by the values specified in the Project Details fields as shown.
 
 .. image:: ../common/images/projects-create-scm-project-branching-emphasized.png
+   :alt: Create New Project page 
 
 Projects have the option to "Allow Branch Override". When checked, project admins can delegate branch selection to the job templates that use that project (requiring only project ``use_role``). 
 
 .. image:: ../common/images/projects-create-scm-project-branch-override-checked.png
+   :alt: Allow Branch Override checkbox option in Project
```

Comment:
We should let readers know that it is actually selected, "Allow Branch Override checkbox option in Project selected"

Reply:
Added messages as suggested.